html-forms/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <form action=>
+    <label for="email" >Email</label>
+    <input type="email" id="email" name="email"/>
+
+
+    <label>Why is coding cool ?</label>
```

Comment:
Every `label` element should have a `for` attribute corresponding to an `id` on the input it is labelling.